src/test/java/core/basesyntax/service/ReaderServiceTest.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.service.impl.ReaderServiceImpl;
+import java.util.List;
+import org.junit.Test;
+
+public class ReaderServiceTest {
+    private static final ReaderService READER = new ReaderServiceImpl();
+
+    @Test(expected = RuntimeException.class)
+    public void reader_read_nonExistentFile_NotOK() {
```

Comment:
Let's follow our common naming convention:
<methodUnderTest>_<state>_<expectedBehavior>
Fix naming in all places